ale/base/data_naif.py

```diff
@@ -235,29 +388,85 @@ def sensor_orientation(self):
 
     @property
     def ephemeris_start_time(self):
+        """
+        Returns the starting ephemeris time of the image. Expects spacecraft_id to
+        be defined. This must be the integer Naif Id code for the spacecraft. Expects
+        spacecraft_clock_start_count to be defined. This must be a floating point number
```

Comment:
The spacecraft_clock_start_count is actually a string. It's just one of those weird things about SPICE.